htdocs/main.inc.php

```diff
@@ -1565,8 +1565,12 @@ function top_htmlhead($head, $title = '', $disablejs = 0, $disablehead = 0, $arr
 
 		if (!defined('DISABLE_FONT_AWSOME')) {
 			print '<!-- Includes CSS for font awesome -->'."\n";
-			print '<link rel="stylesheet" type="text/css" href="'.DOL_URL_ROOT.'/theme/common/fontawesome-5/css/all.min.css'.($ext ? '?'.$ext : '').'">'."\n";
-			print '<link rel="stylesheet" type="text/css" href="'.DOL_URL_ROOT.'/theme/common/fontawesome-5/css/v4-shims.min.css'.($ext ? '?'.$ext : '').'">'."\n";
+
+			$fontawesome_directory = !empty($conf->global->MAIN_FONTAWESOME_DIRECTORY) ? $conf->global->MAIN_FONTAWESOME_DIRECTORY : '/theme/common/fontawesome-5';
```

Comment:
you can use `getDolGlobalString('MAIN_FONTAWESOME_DIRECTORY', '/theme/common/fontawesome-5');`

Reply:
Done, thanks @frederic34